src/ios/OpenTokPlugin.m

```diff
@@ -404,39 +495,54 @@ - (void)sessionDidConnect:(OTSession*)session{
 }
 
 
-- (void)session:(OTSession *)session connectionCreated:(OTConnection *)connection
-{
+- (void)session:(OTSession *)session connectionCreated:(OTConnection *)connection{
     [connectionDictionary setObject: connection forKey: connection.connectionId];
     NSMutableDictionary* data = [[NSMutableDictionary alloc] init];
     NSMutableDictionary* connectionData = [self createDataFromConnection: connection];
     [data setObject: connectionData forKey: @"connection"];
     [self triggerJSEvent: @"sessionEvents" withType: @"connectionCreated" withData: data];
 }
 
-- (void)session:(OTSession *)session connectionDestroyed:(OTConnection *)connection
-{
+- (void)session:(OTSession *)session connectionDestroyed:(OTConnection *)connection{
     [connectionDictionary removeObjectForKey: connection.connectionId];
     NSMutableDictionary* data = [[NSMutableDictionary alloc] init];
     NSMutableDictionary* connectionData = [self createDataFromConnection: connection];
     [data setObject: connectionData forKey: @"connection"];
     [self triggerJSEvent: @"sessionEvents" withType: @"connectionDestroyed" withData: data];
 }
+- (void)sessionDidReconnect:(OTSession*)session {
+    NSLog(@"iOS Session reconnected");
+    NSMutableDictionary* eventData = [[NSMutableDictionary alloc] init];
+    [self triggerJSEvent: @"sessionEvents" withType: @"sessionReconnected" withData: eventData];
+}
+- (void)sessionDidBeginReconnecting:(OTSession*)session {
+    NSLog(@"iOS Session reconnecting");
+    NSMutableDictionary* eventData = [[NSMutableDictionary alloc] init];
```

Comment:
why send an empty dictionary? Can we send null since there is no data associated with this event?